graph/src/data/value.rs

```diff
@@ -170,6 +170,32 @@ impl TryFrom<q::Value> for Value {
     }
 }
 
+impl From<serde_json::Value> for Value {
```

Comment:
I'm confused why there's the need for two `Value`s in the `fork.rs` file, is this really needed or could we just do one JSON transformation?

Reply:
The field types supported by `serde_json::Value` and the `Value` used in the subgraph store are different, so there needs to be a mechanism in which to convert from one to the other. The above `Value` and the `From<serde_json::Value>` provide this mechanism.